src/skeleton/SkeletonDUT.scala

```diff
@@ -32,8 +32,13 @@ class SkeletonDUT(harness: LazyScope)(implicit p: Parameters) extends RocketSubs
   def resetVector: BigInt = 0x80000000L
 
   // set eccBytes equal to beatBytes so we only generate a single memory
-  val main_mem_sram = LazyModule(new TLRAM(AddressSet(resetVector, 0x10000000-1), beatBytes = mbus.beatBytes,
-    ecc = ECCParams(bytes = mbus.beatBytes)))
+  val main_mem_sram = LazyModule(new TLRAM(
+    address = AddressSet(resetVector, 0x10000000-1),
+    beatBytes = mbus.beatBytes,
+    ecc = ECCParams(bytes = mbus.beatBytes),
+    parentLogicalTreeNode = Some(logicalTreeNode),
+    devName = Some("testsocket-main-memory")
```

Comment:
Apparently this thing _has_ to be named `mem` at the moment because this string affects the name of the Verilog instance name and module name of the actual SRAM instance within the TLRAM, which broke this [hardcoded hierarchical path](https://github.com/sifive/api-generator-sifive/blob/2746926805ee00f91aacf883a8bb830c27f69ed2/vsrc/TestDriver.sv#L189) in api-generator-sifive's `TestDriver.sv`:

```systemverilog
    if ($value$plusargs("testfile=%s", testfile))
    begin
      $readmemh(testfile, testHarness.dut.main_mem_sram.mem.mem_ext.ram);
    end
```

I'll fix this up and test again.